README.md

```diff
@@ -219,6 +219,31 @@ const usernameInputElement = getByTestId('username-input')
 > Learn more about `data-testid`s from the blog post
 > ["Making your UI tests resilient to change"][data-testid-blog-post]
 
+#### `toBeInTheDOM`
+
+There are two simple API which extend the `expect` API of jest for making assertions easier.
+The first one `toBeInTheDOM` which allows you to assert whether an element present in the DOM or not
+
+```javascript
+import 'extend-expect' //adds few API to jest's extend
+
+render(<span data-testid="count-value">2</span>)
+expect(queryByTestId('count-value')).toBeInTheDOM()
+expect(queryByTestId('count-value1')).not.toBeInTheDOM()
```

Comment:
Could you add a comment below that says: "Note: for this assertion, make sure you use a `query` function (like `queryByTestId`) rather than a `get` function (like `getByTestId`). Otherwise the get function could throw an error.